ufcc/sampledata.py

```diff
@@ -0,0 +1,16 @@
+r"""UFCC sample data
+======================================================
+:Authors: Daniel P. Ramirez & Besian I. Sejdiu
+:Year: 2022
+:Copyright: MIT License
+"""
+
+import pathlib
+
+class GIRKDataSample:
+    def __init__(self):
+        self.path = pathlib.Path('data').absolute()
```

Comment:
I like the design, but unfortunately, this does not work, at least from my testing.  
We're working with `setuptools`, so you have to look at its documentation on how to treat data files (i.e., load data from the example/data folder **within** the installation directory). For instance: [this link](https://stackoverflow.com/questions/4519127/setuptools-package-data-folder-location)

Reply:
Fixed!!